staking/cli/src/instructions.rs

```diff
@@ -1173,11 +1173,18 @@ pub async fn save_stake_accounts_snapshot(rpc_client: &RpcClient) {
     }
 
     let timestamp = chrono::Utc::now().format("%Y-%m-%d_%H:%M:%S").to_string();
-    let file = File::create(format!("snapshots/snapshot-{}.csv", timestamp)).unwrap();
-    let mut writer = BufWriter::new(file);
-
+    
+    // Create CSV file
+    let csv_file = File::create(format!("snapshots/snapshot-{}.csv", timestamp)).unwrap();
+    let mut csv_writer = BufWriter::new(csv_file);
     // Write the header
-    writeln!(writer, "positions_pubkey,metadata_pubkey,custody_pubkey,custody_authority_pubkey,owner,locked_amount,staked_in_governance,staked_in_ois").unwrap();
+    writeln!(csv_writer, "positions_pubkey,metadata_pubkey,custody_pubkey,custody_authority_pubkey,owner,locked_amount,staked_in_governance,staked_in_ois").unwrap();
+
+    // Create JSON file
```

Comment:
Rather than always writing out both files, IMO we should have a CLI option to specify if you want json instead of csv